server/cmwell-controller/src/pack/application.conf

```diff
@@ -2,4 +2,15 @@ updatesTlog.name=UpdateTLog
 uuidsTlog.name=UuidsTLog
 
 cmwell.clusterName = "CmwellCluster"
-cmwell.grid.clusterName = ${cmwell.clusterName}
\ No newline at end of file
+cmwell.grid.clusterName = ${cmwell.clusterName}
+
+akka {
```

Comment:
Are we sure that `application.conf` is not packaged with jar?
Because thecontroller is also used a s a library.
In case it is packaged inside the jar, in the controller only, when we create akka's objects, we should either load `controller.conf`, via `LoggerFactory.load("controller")` or, nest all these properties under `cmwell.ctrl`, and extract the akka subtree from config in code.
And if it is left as is, `application.conf` will collide (i.e: merged & hard to reason about) with other application.conf files when ctrl is used as library.